shell/platform/android/io/flutter/embedding/android/FlutterFragment.java

```diff
@@ -1071,6 +1072,12 @@ public void onAttach(@NonNull Context context) {
   @Override
   public void onCreate(@Nullable Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
+    if (savedInstanceState != null) {
+      boolean frameworkHandlesBack =
```

Comment:
No need to change the pr now but when we have a second variable then all of this code should go into a "save everything" and "restore everything" methods.